device-types/Palo Alto/PA-5450.yaml

```diff
@@ -0,0 +1,44 @@
+---
+manufacturer: Palo Alto
+model: PA-5450
+slug: pa-5450
+part_number: PA-5450
```

Comment:
Please correct me if I am wrong, but this appears to be the 5450 chassis device. I believe the chassis part number is `PAN-PA-5450-AC-SYS`, is this correct? If so please update the `slug` and `part_number`

Reply:
That is correct. It is a hardware bundle that includes one network card, one management card, and one data processor card and power supplies.  I will make the change.